.github/workflows/master_build_merge.yml

```diff
@@ -1087,7 +1087,9 @@ jobs:
             requiredInputs="1"
             validateInputs "$requiredInputs" "$@" 
             awsConfigureProdAssets
-            cd tessell;
+            if [ ! -f "Makefile" ]; then
```

Comment:
Why is this change needed?

Reply:
It's because the makefile in readiness-lib is present at the top level (whereas in the tessell-task-handler the the file is present inside tessell dir